doc/Developing.md

```diff
@@ -44,14 +44,29 @@ For Windows instructions [click here](#building-on-windows).
      - [Root privileges errors](#root-privileges-errors)
 
 ## Prerequisites
+
  - Node.js `>= 10.2.0`.
    - Preferably, **use** version `10.15.3`, it has the [active LTS](https://github.com/nodejs/Release).
    - Node.js `11.x` is untested.
  - [Yarn package manager](https://yarnpkg.com/en/docs/install) v1.7.0
  - git (If you would like to use the Git-extension too, you will need to have git version 2.11.0 or higher.)
 
-[nvm](https://github.com/creationix/nvm) is recommended to easily switch between
-Node.js versions.
+Some additional tools and libraries are needed depending on your platform:
+
+- Linux
```

Comment:
We will need to confirm that these additional tools and libraries are solely required on Linux.
The way it's worded states that only Linux requires these tools and libraries which may or may not be true.

Reply:
I only have a Linux at hand and only troubleshooted Linux issues from others, it would take someone knowledgeable on MacOS, which I am not.